README.md

```diff
@@ -1,95 +1,69 @@
 
-    cp config/jsbin_config.example.yaml config/jsbin_config.yaml
+### Running with Ruby
 
-And then you will need to populate the `api_key` property in that file with a JSBin API key.
+Using Ruby directly (2.7.0) you must first run the following command:
 
-The creation of code content for JSBin is further documented in our
-[document formatting guide](content/client-lib-development-guide/documentation-formatting-guide.textile)
-which gets published
-[here](https://docs.ably.io/client-lib-development-guide/documentation-formatting-guide/#code-blocks).
-The code that is published to JSBin can be found in
-[content/code/](content/code/).
-
-## Forking and running locally
-
-* Fork the repository at https://github.com/ably/docs
-* Clone a local version of your forked repo `git clone https://github.com/[you]/docs` and `cd` into it
-* Checkout the master branch: `git checkout master`
-* Create your feature or fix branch and check it out: `git checkout -b [your-branch]`
-* `bundle install` to install the necessary gems, including `nanoc`
-
-At this point you can either run the repo directly on your machine or via Docker.
+* `bundle install` 
 
-### Running directly with Ruby
+This installs the necessary gems, including `nanoc`. You can then run the web server by running the following command:
 
-If you have Ruby and are comfortable working in a Ruby environment as follows:
+* `bundle exec nanoc live -p 4000` 
 
-* `bundle exec nanoc live -p 4000` to start up a web server at <http://localhost:4000>, build the static site and view changes live (well near-live as Nanoc is slow, so monitor the output as you make changes)
+This starts up a web server at <http://localhost:4000>, builds the static site and allows you to view changes live.
 
 _For those who use [ASDF](https://github.com/asdf-vm/asdf) or compatible tooling to manage their Ruby runtime versions, we have included a [`.tool-versions`](.tool-versions) file._
 
-### Running with Docker and Docker compose
-
-If you would prefer to use Docker, then you can run the web server as follows:
+## Pull requests
 
-* `docker-compose up`
+1. Assuming you have followed the steps above, prior to making a PR, make your changes locally.
+2. Commit your changes, push your branch and send us a pull request.
+3. All pull requests will automatically launch a static review site that will be linked to at the bottom of the PR.
 
-_This has been tested with Docker version 2.3.0.3. See [Docker installation instructions](https://docs.docker.com/get-docker/)._
-
-## Running on Heroku
+## Publishing to JSBin
 
-This repo will automatically run on Heroku, but relies on the following buildpacks (see https://github.com/ably-forks/heroku-buildpack-nanoc):
+If you will be creating or modifying code within this repository that is to be uploaded automatically
```

Comment:
I think this is a bit ambiguous.  There is code all over the place in our repository.  JSBin only applies to code in the code folder.  I also think we should link to the JSBin repo with information on how that works as there are docs in there too.